### ISSUE TITLE
Order of Waypoint items

### ISSUE THREAD

**Issue by tamielbr** [scholarslab/Neatline#270]
Would it be possible to create a feature to organize the Waypoint list display in alphabetical order (it can only be done manually now, which is complicated with dozens of items).

**Comment by davidmcclure** [comment]
Hey @tamielbr - Yeah, I can see how that would be useful. With this kind of functionality, the question is always how to do it in a sustainable that doesn't add undue maintenance overhead to the code. Would you want the waypoints to _always_ be displayed alphabetically in all exhibits, or just in one or two individual exhibits?

**Comment by tamielbr** [comment]
I agree it might not always be the case. I think as you get to hundreds of items (we are working on a map of cultural instalations in our city), it gets complicated to sort them by hand. It would be very useful, but perhaps it can be measured against other demands...for now we'll do it by hand!
